blockly/src/utils/LevelUtils.java

```diff
@@ -41,4 +42,41 @@ public static boolean canSee(
     }
     return true;
   }
+
+  /**
+   * Checks if a given coordinate is walkable.
+   *
+   * @param coord The coordinate to check.
+   * @return true if the tile at the given coordinate is accessible, false otherwise.
+   */
+  public static boolean isWalkable(Coordinate coord) {
+    Tile tile = Game.tileAT(coord);
+    if (tile == null) {
+      return false;
+    }
+    return tile.isAccessible();
+  }
+
+  /**
+   * Retrieves a list of walkable neighboring coordinates for a given coordinate.
+   *
+   * @param coord The coordinate for which to find walkable neighbors.
+   * @return A list of walkable neighboring coordinates. Returns an empty list if the given
+   *     coordinate is not walkable.
+   */
+  public static List<Coordinate> walkableNeighbors(Coordinate coord) {
+    if (!isWalkable(coord)) {
+      return List.of();
+    }
+
+    List<Coordinate> neighbors =
+        List.of(
+            coord.add(new Coordinate(1, 0)), // Right
+            coord.add(new Coordinate(-1, 0)), // Left
+            coord.add(new Coordinate(0, 1)), // Down
+            coord.add(new Coordinate(0, -1)) // Up
```

Comment:
hmmm, irgendwie kommt mir das so bekannt vor. ist das nicht schon in einer der wenigen `direction`-klassen formuliert?

Reply:
Jup, werde ich wohl noch ändern müssen auf Direction (irgendeine von den vielen :D)